nns/nns_contract.go

```diff
@@ -336,17 +345,35 @@ func Register(name string, owner interop.Hash160, email string, refresh, retry,
 	} else {
 		updateTotalSupply(ctx, +1)
 	}
-	ns := NameState{
+	saveDomain(ctx, name, email, refresh, retry, expire, ttl, owner)
+	updateBalance(ctx, []byte(name), owner, +1)
+	postTransfer(oldOwner, owner, []byte(name), nil)
+	return true
+}
+
+//nolint:unused
+func saveCommitteeDomain(ctx storage.Context, name, email string) {
+	putTLD(ctx, makeTLDKey(name))
+
+	// TODO: values from NeoFS ADM, check
+	const (
+		refresh = 3600
+		retry   = 600
+		expire  = 10 * 365 * 24 * 60 * 60 // 10 years
+		ttl     = 3600
+	)
+	var committeeOwner interop.Hash160
+	saveDomain(ctx, name, email, refresh, retry, expire, ttl, committeeOwner)
+}
+
+func saveDomain(ctx storage.Context, name, email string, refresh, retry, expire, ttl int, owner interop.Hash160) {
```

Comment:
Ditto here and below, we'd better keep all the methods documented, NNS is being changed too often.

Reply:
tried to clarify what's going on